openhtf/__init__.py

```diff
@@ -192,14 +192,14 @@ def configure(self, **kwargs):
     # side effects.
     create_arg_parser(add_help=True).parse_known_args()
     logs.setup_logger()
-    for key, value in kwargs.iteritems():
+    for key, value in kwargs.items():
       setattr(self._test_options, key, value)
 
   @classmethod
   def handle_sig_int(cls, *_):
     if cls.TEST_INSTANCES:
       _LOG.error('Received SIGINT, stopping all tests.')
-      for test in cls.TEST_INSTANCES.values():
+      for test in list(cls.TEST_INSTANCES.values()):
```

Comment:
don't need to list() something in for, since it accepts iterators too. Another unnecessary 2to3 change